data/low_income_thresholds.json

```diff
@@ -154,6 +154,21 @@
       }
     }
   },
+  "IL": {
+    "default": {
+      "source_url": "https://dceo.illinois.gov/communityservices/homeweatherization.html",
+      "thresholds": {
```

Comment:
I wasn't sure if this was still valuable to keep if no incentives had defined income restrictions, but I wanted to check. Slack thread: https://rewiringameri-g3x1100.slack.com/archives/C04E4THEGMT/p1708466841224109